src/server/wtr-config.js

```diff
@@ -0,0 +1,145 @@
+import { argv } from 'node:process';
+import { defaultReporter } from '@web/test-runner';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+
+const DEFAULT_PATTERN = type => `./test/**/*.${type}.js`;
+const DEFAULT_VDIFF = false;
+const ALLOWED_BROWSERS = ['chromium', 'firefox', 'webkit'];
+
+export class WTRConfig {
+
+	#cliArgs;
+	#requestedBrowsers;
+
+	constructor(cliArgs) {
+		this.#cliArgs = cliArgs || [];
+		this.#requestedBrowsers = this.#cliArgs?.toString().match(new RegExp(ALLOWED_BROWSERS.join('|'), 'gi'));
+	}
+
+	get visualDiffGroup() {
+		return {
+			name: 'vdiff',
+			files: this.pattern('vdiff'),
+			browsers: this.getBrowsers(['chromium']),
+			testRunnerHtml: testFramework =>
+				`<html lang="en">
+					<head>
+						<link rel="preload" href="https://s.brightspace.com/lib/fonts/0.5.0/assets/Lato-400.woff2" as="font" type="font/woff2" crossorigin>
+						<link rel="preload" href="https://s.brightspace.com/lib/fonts/0.5.0/assets/Lato-700.woff2" as="font" type="font/woff2" crossorigin>
+						<style>
+							html {
+								font-size: 20px;
+							}
+							body {
+								background-color: #ffffff;
+								margin: 0;
+								padding: 30px;
+							}
+							body[data-theme="dark"] {
+								background-color: #000000;
+							}
+							body[data-theme="translucent"] {
+								background: repeating-linear-gradient(45deg, #606dbc, #606dbc 10px, #465298 10px, #465298 20px);
+							}
+						</style>
+						<script type="module" src="./components/typography/typography.js"></script>
```

Comment:
Oh interesting, what to do with this... 🤔

We definitely don't want to maintain a duplicate of this, maybe we do the same thing as @dlockhart's PR and come up with a "minimum" version?

I wonder if we could make `core` a peer dependency to avoid the circular dependency.  But then what does that do when you try to use this _in_ `core` 😬

Reply:
Oof. Good question. Being peer dependency seems more promising. I'll try setting it up that way and see what happens.

Reply:
Good catch. If that doesn't work, having minimal typography should be pretty doable.